github_connector/models/github_organization_serie.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class GithubOrganizationSerie(models.Model):
+    _name = 'github.organization.serie'
```

Comment:
Awesome, thanks for the explanation! I wonder if we should maybe call them `release` instead, because they are release branches?

If we stick with series though- `series` is both a singular and plural form of the word, so this would be `github.organization.series`. English is stupid sometimes 😑 .